test/functional/testlib.bash

```diff
@@ -1175,7 +1175,9 @@ test_setup() {
 # Default test_teardown
 test_teardown() {
 
-	destroy_test_environment "$TEST_NAME"
+	if [ "$DEBUG_TEST" != true ]; then
```

Comment:
++ for this commit. That's really useful.

Reply:
This should have a corresponding update to the test readme.

Reply:
that is true @matthewrsj, good catch. I'll add that to the commit.